test/e2e/common/secrets.go

```diff
@@ -134,6 +136,66 @@ var _ = ginkgo.Describe("[sig-api-machinery] Secrets", func() {
 		secret, err := createEmptyKeySecretForTest(f)
 		framework.ExpectError(err, "created secret %q with empty key in namespace %q", secret.Name, f.Namespace.Name)
 	})
+
+	ginkgo.It("should patch a secret", func() {
+		ginkgo.By("creating a secret")
+
+		secretTestName := "test-secret-" + string(uuid.NewUUID())
+
+		// create a secret in namespace default
+		_, err := f.ClientSet.CoreV1().Secrets("default").Create(&v1.Secret{
```

Comment:
create objects in the test namespace (f.Namespace.Name), not in a shared namespace

Reply:
Thanks for pointing this out, that's left over from the mock test

Reply:
Updated in b4fb46409d1